src/routes/api/editor/suggestions/+server.ts

```diff
@@ -1,80 +1,18 @@
-import { OPENAI_API_KEY } from '$env/static/private';
-import { trackEvent } from '$lib/server/amplitude.js';
-import { editors } from '$lib/server/editors/editors.js';
-import { OpenAILLM } from '$lib/server/llms/openai.js';
+import { getSuggestions } from '$lib/server/editors/getSuggestions.js';
```

Comment:
nit: ts

Reply:
it's always js I think

Reply:
you're right apparently my brain was not ready for reviews yet